lib/decimal.ex

```diff
@@ -49,6 +49,29 @@ defmodule Decimal do
 
   import Kernel, except: [abs: 1, div: 2, max: 2, min: 2, rem: 1, round: 1]
 
+  @typedoc """
+  The coefficient of the power of `10`.  Non-negative because the sign is stored separately in `sign`.
+
+  * `non_neg_integer` - a real number
+  * `:qNaN` - a quiet NaN was produced by a previous operation.  Quiet NaNs propagate quietly, unlike signaling NaNs
+     that return errors (based on the `Decimal.Context`).
+  * `:sNaN` - signalling NaN that indicated an error occured that should stop the next operation with an error
+     (based on the `Decimal.Context`).
+  * `:inf` - infinity
```

Comment:
Is this so they standout more in the literal `@doc`?  I'm not seeing a change in the rendered HTML.